pkg/tsdb/interval.go

```diff
@@ -6,25 +6,56 @@ import (
 )
 
 var (
-	defaultRes  int64         = 1500
-	minInterval time.Duration = 1 * time.Millisecond
-	year        time.Duration = time.Hour * 24 * 365
-	day         time.Duration = time.Hour * 24 * 365
+	defaultRes         int64         = 1500
+	defaultMinInterval time.Duration = 1 * time.Millisecond
+	year               time.Duration = time.Hour * 24 * 365
+	day                time.Duration = time.Hour * 24 * 365
```

Comment:
Why `day` definition has same value as and `year`? Is it not typo ?

Reply:
Good catch! :)

Fixed in my last commit. Good thing it was only used for formatting.